pkg/asset/manifests/tectonic.go

```diff
@@ -55,6 +69,7 @@ func (t *Tectonic) Generate(dependencies asset.Parents) error {
 	}
 
 	assetData := map[string][]byte{
+		"99_aws-creds-secret-and-reader-role.yaml": applyTemplateData(content.AwsCredsSecretAndReaderRole, templateData),
```

Comment:
Only want to create this if cloud-provider is "aws" so.. have to figure that out.